parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -134,9 +134,17 @@ ensureDHCPv6
 configPrivateClusterHosts
 {{- end}}
 
+{{- if ShouldConfigTransparentHugePage}}
+configureTransparentHugePage
+{{- end}}
+
 ensureKubelet
 ensureJournal
 
+{{- if ShouldConfigSysctl}}
+configureSysctl
+{{- end}}
+
```

Comment:
shall we do this before ensureKubelet?

Reply:
Now use a new func ensureSysctl and put before ensureKubelet